app.js

```diff
@@ -79,7 +78,7 @@ app.use((err, req, res, next) => {
     }
   });
   if (isDevelopment) {
-    next(err);
+    // next(err);
```

Comment:
@augustineezinwa how come this is commented out? I suggest we take it out - along with the if block - if it is no longer need.